apps/webapp/Dockerfile.slim

```
@@ -1,61 +1,57 @@
-ENV PATH="$PNPM_HOME:$PATH"
-RUN apk add --no-cache libc6-compat && \
-    corepack enable
+# Alpine image
+FROM node:20-alpine3.21 AS alpine
+RUN apk update
+RUN apk add --no-cache libc6-compat openssl
 
-WORKDIR /app
+# Setup pnpm and turbo on the alpine base
+FROM alpine as base
+RUN npm install pnpm turbo --global
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)